k8s/01/README.md

```diff
@@ -9,7 +9,7 @@ Check the external IP of the loadbalancer that was deployed with the app with `k
 to find out how to connect to the service.
 
 Note: When you use minikube you have to run `minikube tunnel` to connect to you loadbalancer. 
-The service will then be reachable through http://localhost:4567/ .
```

Comment:
> the external IP from the tunnel command 

Not sure what you mean with this. For me the tunnel command does not give me an external IP, but will assign `127.0.0.1` as external IP when i inspect with `kubectl  get service`. Is this different for you?

Reply:
Yes, it assigned some `10.x.x.x` IP, not sure why. It was definitely not localhost. Had to use that IP address to connect.